CHANGELOG.md

```diff
@@ -19,12 +19,18 @@ _None_
 * Images: new dot-syntax template, use `dot-syntax-swift3` or `dot-syntax` (for Swift 2.3).  
   [David Jennes](https://github.com/djbe)
   [#206](https://github.com/AliSoftware/SwiftGen/pull/206)
+* Reworked the "dot-syntax" and "structured" templates to use the new `define` and `call` tags, which greatly simplifies the templates, and also removes the limitation of 5-level deep structures.  
```

Comment:
&lt;nitpicking mode ON&gt;
I know the 80-columns-limit thing is a bit outdated, but still, don't hesitate to line-wrap the CHANGELOG entries in the raw markdown (the CRLF will be ignored once rendered unless the previous line ends with two spaces, so…)
&lt;nitpicking mode OFF&gt;